models/model_choice.py

```diff
@@ -141,6 +141,28 @@ def net(net_params, num_channels, inference=False):
             in_channels=num_bands,
             classes=num_channels,
             activation=None)
+    elif model_name == 'spacenet_unet_efficientnetB5_pretrained':
```

Comment:
I think we can create a markdown file to document potential models we can use with smp. It would be counterproductive to define so many models as code here since they can easily be called by defining smp parameters.

Reply:
Good,  I had the intention to make some MD in some folder for a better documentation, I dont know when I will have the time but yes.